src/qibo/backends/abstract.py

```diff
@@ -376,6 +376,13 @@ def calculate_singular_value_decomposition(self, matrix):  # pragma: no cover
         """Calculate the Singular Value Decomposition of ``matrix``."""
         raise_error(NotImplementedError)
 
+    @abc.abstractmethod
+    def calculate_jacobian_matrix(
```

Comment:
I was following the same name pattern of other backend methods like `calculate_eigenvalues`, `calculate_matrix_power`, etc.

Maybe it's worth to open another PR to rename them all.

Reply:
> I was following the same name pattern of other backend methods like `calculate_eigenvalues`, `calculate_matrix_power`, etc.

Ok, fine, keep going.

> Maybe it's worth opening another PR to rename them all.

In case you consider it meaningful, let's open an issue for that, and accurately schedule for the future.
Bulk changing backend functions (even if not the most frequently used, like `execute_circuit`) is definitely a breaking change. Better to plan it carefully.

Reply:
I don't consider it meaningful TBH

Reply:
> I don't consider it meaningful TBH

I would have chosen different names. In general, the simplest non-ambiguous ones. But I prefer avoiding breaking.

To be fair, I personally consider `jacobian` different from `calculate_eigenvalues`, since the Jacobian requires a dedicated execution (while the other one is about results manipulation).
But this means to me that the Jacobian will be propagated to `qibo-core`, while `calculate_eigenvalues` will be split out the `qibo-core` backends.

Until that time comes, do whatever you wish.